.github/workflows/stress_test_acceptance_test.yml

```diff
@@ -0,0 +1,77 @@
+# This workflow is used to run the acceptance tests multiple times.
+# It's not used in the CI, but is used to test the falkiness of an acceptance test.
+
+name: Stress Test (Acceptance Test)
+
+permissions: read-all
+on:
+  workflow_dispatch:
+    inputs:
+      num_runs:
+        description: 'Number of times to run the acceptance tests (Max: 128).'
+        required: true
+        default: '10'
+        type: string
+      test_suite:
+        description: 'Acceptance test to run. (e.g. "interested-donor/make-a-donation")'
+        required: true
+        type: string
```

Comment:
Please validate these inputs to make sure we block any code injection attacks. As a second line of protection, please also use GitHub's [recommended mitigation strategies](https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks) (intermediate environment variables are probably your best bet) when using these user-provided, and therefore potentially malicious, inputs

Reply:
Done.